bin/node/src/commands/node.rs

```diff
@@ -200,6 +201,12 @@ impl NodeCommand {
     /// Run the Node subcommand.
     pub async fn run(self, args: &GlobalArgs) -> anyhow::Result<()> {
         let cfg = self.get_l2_config(args)?;
+
+        // If metrics are enabled, initialize the global cli metrics.
+        if args.metrics.enabled {
+            init_rollup_config_metrics(&cfg);
+        }
```

Comment:
Think this could be done inline since there's no return - not a big deal tho

```rust
        args.metrics.enabled.then(|| init_rollup_config_metrics(&cfg));
```